GVFS/GVFS.Mount/InProcessMountVerb.cs

```diff
@@ -53,7 +53,15 @@ public InProcessMountVerb()
                 MetaName = "Enlistment Root Path",
                 HelpText = "Full or relative path to the GVFS enlistment root")]
         public string EnlistmentRootPathParameter { get; set; }
-        
+
+        [Value(

```

Comment:
Nit: I would change this to an `Option` instead of a `Value`. When seeing the command line args from a user's procmon, for example, it'll be a lot easier to understand `--initiated-by-clone` rather than just `true` in the args list.

Though given your other discussion about always logging an enlistmentId, is this going to possibly just go away?

Reply:
@sanoursa enlistmentId is a one to many relationship with mounts, so adding that alone doesn't give us a simple query to find all the mount failures due to Clone.  However mountId is a one-to-one relationship so that would give us an easy lookup.  I had another look at where we were setting these fields and landed on a different approach that hopefully is less invasive.  If we're ok with my latest changes i'll update the .md file accordingly with the new queries.